glue-codes/simulink/src/FAST_SFunc.c

```diff
@@ -204,6 +204,7 @@ static void mdlInitializeSizes(SimStruct *S)
        if (checkError(S)) return;
 
        FAST_Sizes(&iTurb, InputFileName, &AbortErrLev, &NumOutputs, &TMax, &dt, &ErrStat, ErrMsg, ChannelNames, &TMax, InitInputAry);
```

Comment:
Should this line be dropped?

Reply:
yeah, it looks like it got committed strangely. Let me fix that.